src/platform/common/utils.node.ts

```diff
@@ -29,45 +23,3 @@ export async function tryGetRealPath(expectedPath: Uri): Promise<Uri> {
         return expectedPath;
     }
 }
-
-export async function calculateWorkingDirectory(
-    configService: IConfigurationService,
-    workspace: IWorkspaceService,
-    fs: IFileSystem,
-    resource: Resource
-): Promise<Uri | undefined> {
```

Comment:
This code is not common code at all, its very specific to kernels
We have `settings.notebookFileRoot` being used here, hence this shouldn't be in the `src/platform/common` directory.